### PR TITLE
Add deque best practices to our ruleset 🍖

### PR DESCRIPTION
**Pull request by coltborg**
#### ✍️ Description
Add the deque best practice rule set to our a11y tests.

- https://dequeuniversity.com/rules/axe/4.8/#best-practices-rules

### CODE REVIEW

**Comment by coltborg** [comment]
Yay, it failed! Because we are violated the "no content outside of regions" rule. So we need to get the "Go back links" into a region for the tests to pass.

**Comment by coltborg** [comment]
Yes! The go back region issue is fixed!

There are now 2 a11y tests failing. Look like there are two screens that do not have an `h1`. This is an issue with screen readers. I'll add a fix to the PR today.

File: src/main/resources/templates/gcc/activities-ed-program-name.html [hunk 1]
```diff
@@ -1,29 +1,17 @@
...
-</div>
-<th:block th:replace="~{fragments/footer :: footer}"/>
-</body>
-</html>
+<th:block
+        th:replace="~{fragments/screens/screenWithOneInput ::
+  screenWithOneInput(
+    title=#{activities-add-ed-program-name.header},
+    iconFragment=~{fragments/gcc-icons :: mortarboard},
+    header=#{activities-add-ed-program-name.header},
```
Comment (enyia21) [inline src/main/resources/templates/gcc/activities-ed-program-name.html:6]: The header for this field is missing the `*` at the end to that alerts users that this field is required.  Other than that this is good to go

image

**Review by enyia21 — APPROVED** [review]
I still think this is ready to ship because I don't think the screen change needs to be fixed in this PR but I wanted to flag the copy difference.  

The `Go Back` link worked great.  
All a11y tests passed.

File: src/main/resources/templates/gcc/activities-ed-program-name.html (hunk 1 above)
Comment (coltborg) [reply src/main/resources/templates/gcc/activities-ed-program-name.html:6]: Added the `*`.

**Pull request merged**